q2_types/sample_data/tests/test_transformer.py

```diff
@@ -30,11 +30,24 @@ def test_pd_series_to_alpha_diversity_format(self):
 
     def test_alpha_diversity_format_to_pd_series(self):
         filename = 'alpha-diversity.tsv'
-        input, obs = self.transform_format(AlphaDiversityFormat, pd.Series,
-                                           filename)
-        input = pd.Series.from_csv(str(input), sep='\t', header=0)
+        _, obs = self.transform_format(AlphaDiversityFormat, pd.Series,
+                                       filename)
 
-        assert_series_equal(input, obs)
+        exp_index = pd.Index(['Sample1', 'Sample4'], dtype=object)
+        exp = pd.Series(['0.9709505944546688', '0.7219280948873623'],
```

Comment:
Thanks for the details! For safety I'd prefer to revert to the old behavior of keeping the values as floats -- ideally the unit tests / busywork would catch any issues but I don't have confidence in the existing tests (some are missing, others are very basic/lenient).